### PR TITLE
Redesign decryption UI

### LINKED ISSUE #48: Make hyperlinks in decrypted text clickable!

### PR DESCRIPTION
**Pull request by keur**
* Import bootstrap into the iFrame
* Change text display from textarea to div
* Make links clickable (requires escaping input)
* Keep hidden textarea for copying text

This pull request closes #48.

### Submitter Checklist

- [x] Tested on Firefox (`npm run firefox`)
- [x] Tested on Chromium (`npm run chromium`)

### CODE REVIEW

File: public/html/decrypt.html [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <head>
   <meta charset="utf-8">
   <title>Babble Decryption</title>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
```
Comment (yvbbrjdr) [inline public/html/decrypt.html]: Use the Bootstrap CSS in the public directory?

Comment (keur) [reply public/html/decrypt.html]: I used the same source as in options. We should probably change it there too then.

**Comment by keur** [comment]
@yvbbrjdr Can you please try again? The regex failed to compile in Firefox because lookbehinds are not yet supported. Should work now.

File: public/html/decrypt.html (hunk 1 above)
Comment (keur) [reply public/html/decrypt.html]: I was not able to get it working.

**Comment by yvbbrjdr** [comment]
@keur `https://github.com/yvbbrjdr` will be decrypted to `https://github.com/;yvbbrjdr`.

**Comment by keur** [comment]
@yvbbrjdr fixed and pushed

**Pull request merged**